api/handlers_managedinstances.go

```diff
@@ -0,0 +1,186 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/YaleSpinup/spot-api/apierror"
```

Comment:
migrate over to the shared `apierror` package.  it should be a drop-in replacement.